seaworthy/containers/postgresql.py

```diff
@@ -0,0 +1,94 @@
+from seaworthy.logs import output_lines
+from .base import ContainerBase
+
+
+class PostgreSQLContainer(ContainerBase):
+    DEFAULT_NAME = 'postgresql'
+    DEFAULT_IMAGE = 'postgres:alpine'
+    # The postgres image starts up PostgreSQL twice--the first time to set up
+    # the database and user, and the second to actually run the thing.
+    DEFAULT_WAIT_PATTERNS = (
+        r'database system is ready to accept connections',
+        r'database system is ready to accept connections',)
+
+    DEFAULT_DATABASE = 'database'
+    DEFAULT_USER = 'user'
+    DEFAULT_PASSWORD = 'password'
+
+    def __init__(self,
+                 name=DEFAULT_NAME,
+                 image=DEFAULT_IMAGE,
+                 wait_patterns=DEFAULT_WAIT_PATTERNS,
+                 database=DEFAULT_DATABASE,
+                 user=DEFAULT_USER,
+                 password=DEFAULT_PASSWORD,
+                 **kwargs):
+        """
+        :param database: the name of a database to create at startup
+        :param user: the name of a user to create at startup
+        :param password: the password for the user
+        """
+        super().__init__(name, image, wait_patterns, **kwargs)
+
+        self.database = database
+        self.user = user
+        self.password = password
+
+    def base_kwargs(self):
+        return {
+            'environment': {
+                'POSTGRES_DB': self.database,
+                'POSTGRES_USER': self.user,
+                'POSTGRES_PASSWORD': self.password,
+            },
+            'tmpfs': {'/var/lib/postgresql/data': 'uid=70,gid=70'},
```

Comment:
Wondering if these `tmpfs` things belong here or if they should be optional 🤔

Reply:
Nah, I think this is fine. If someone specifically wants the data written to disk they can override the container definition.